core/systems/input/managed_gamepad.gd

```diff
@@ -121,6 +122,8 @@ func open(path: String) -> int:
 	if not "--disable-grab-gamepad" in OS.get_cmdline_args():
 		phys_device.grab(true)
 
+	# Store value of get_phys() so this can be reidentified if disconnected.
+	phys = phys_device.get_phys()
```

Comment:
Aren't we already setting this in `phys_path`?

Reply:
phys_path is poorly named. that is the /dev/input/eventX character file. get_phys() returns the USB or PCI header connection address.

Reply:
This is done because it might detect as a different event on re-connection. That does remind me though... I need to check for that and re-associate that new path properly.